pvr.iptvsimple/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.iptvsimple"
-  version="20.3.1"
+  version="20.3.2"
```

Comment:
As you are adding functionality the minor version should be increased. So it should be `20.4.0`.